### DISCUSSION TITLE
TBC PvP and Heroic Dungeon gems

### DISCUSSION THREAD

**Discussion by Tereneckla** [chromiecraft/chromiecraft#4232] [Suggestions]
There are 6 epics gems available for honor in tbc
https://wowgaming.altervista.org/aowow/?item=28118
https://wowgaming.altervista.org/aowow/?item=28119
https://wowgaming.altervista.org/aowow/?item=28120
https://wowgaming.altervista.org/aowow/?item=28123
https://wowgaming.altervista.org/aowow/?item=28362
https://wowgaming.altervista.org/aowow/?item=28363
These were BoP and Unique-Equipped in tbc but are now neither of those.

Then there are the epic pvp gems available for arena points which were added in 2.4
https://wowgaming.altervista.org/aowow/?item=38545
https://wowgaming.altervista.org/aowow/?item=38546
https://wowgaming.altervista.org/aowow/?item=38547
https://wowgaming.altervista.org/aowow/?item=38548
https://wowgaming.altervista.org/aowow/?item=38549
https://wowgaming.altervista.org/aowow/?item=38550

Then there also are all the epic dungeon gems added in 2.1 (not gonna list them now, way to long list)
These also were unique equipped until 2.4
![image](https://user-images.githubusercontent.com/50233983/196045034-ce5d66f9-7003-42e2-aebf-93743516c1ae.png)
https://www.wowhead.com/tbc/item=30547/luminous-fire-opal#comments:id=264691

Access to these gems should be restricted in the same way as they were in original tbc to:
a. Let jewelcrafters actually sell their gem
b. Not have additional unwanted stats that make tuning even harder

**Comment by balleny** [comment]
the **epic dungeon gems** are of the same quality as the **rare jewelcrafter gems**
until we get access to the epic jewelcrafter gems, they are essentially a drop in replacement only with the caveat of being bind on pickup.

The pvp gems should be gated by content, as has the previous pvp gear. This should and surely will apply also for the items that can be bought for badges.

altering the dungeon drop gems to unique seem very unlikely, hence i wouldnt expect that to happen.

so cutting it short, what I expect to happen:
- no change to dungeon epic gems
- pvp epic gems accessible with ZA release (or if release orders are changed: between Black Template and Sunwell release)
- access to epic gems through badges same as pvp epic gems, only late in the game

**Reply by Tereneckla** [reply]
Just want to say that the epic dungeon gems are not the same quality as the rare jewelcrafter gems. They are slightly above them
Craft 5 SP 4 Int https://wowgaming.altervista.org/aowow/?item=24060
Dungeon 6 SP 4 Int https://wowgaming.altervista.org/aowow/?item=30547
Epic Craft 6 SP 5 Int https://wowgaming.altervista.org/aowow/?item=32219

So they are not a replacement

**Reply by balleny** [reply]
true, i as a bit quick to judge bc i looked at one stat only

**Comment by Amrx2** [comment]
Hi Tereneckla, apologies about the length of time needed to respond here. We have been looking at these as a team, not only the issue with Jewelcrafters but other professions as well. We are just bashing out the final details and expect more information on this on an official 'how its going to work in TBC' post coming soon from us all.

Have a good day,
Construct
